src/dom.js

```diff
@@ -111,6 +111,9 @@ export function closest(element, callback) {
  * @return {?Element}
  */
 export function closestByTag(element, tagName) {
+  if (element.closest) {
```

Comment:
Hmm. I ran this test it was remarkably slower than parent walk. Not sure if this is a good advise...